ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -239,7 +232,7 @@
             provider-id native-id request)))
 
        ;; Generic documents are by pattern: /providers/{prov_id}/{concept-type}/{native_id}
-       (context ["/:concept-type" :concept-type generate-generic-concept-types-reg-ex] [concept-type]
+       (context ["/:concept-type" :concept-type (re-pattern common-generic/generic-concept-types-reg-ex)] [concept-type]
```

Comment:
Should this actually be calling the `plural-generic-concept-types-reg-ex` function. The `generate-generic-concept-types-reg-ex` function we are removing here is pluralizing the regex list?

Reply:
My understanding was that the ingest routes use the singular form, and the the search and bootstrap routes use the plural form.

Reply:
Done, ingest is plural